components/reddit/reddit.app.js

```diff
@@ -0,0 +1,189 @@
+const axios = require("axios");
+const get = require("lodash.get");
+module.exports = {
+  type: "app",
+  app: "reddit",
+  propDefinitions: {
+    subreddit: {
+      type: "string",
+      label: "Subreddit",
+      description: "The subreddit you'd like to watch.",
+      optional: false,
+      /* #region  subreddit prop's async options method     */
```

Comment:
Are you using this `#region` comment in a specific way?

Reply:
Yes, I have an add on that allows folding on visual studio code on #region/#endregion pre processor directivoes. I will inquire about this on the chat.